### PR TITLE
CSS Two Column Layout - Ryan Bae

### PR DESCRIPTION
**Pull request by rjbae**
Screen Shot 2022-05-12 at 7 16 48 PM

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          <p class="header-text">
+            We've dedicated thousands of man hours to making healthcare institutions
+            in the Helium network run 10x more efficiently and deliver an unparalleled
+            quality of care.
+          </p>
+        </header>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-third">
```
Comment (Zhiwutian) [inline css-two-column-layout/index.html:27]: the update I would recommend is creating a `column-half` class and wrap these two columns in to to contain all the elements for each of the `two columns`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking great! small update needed

**Comment by rjbae** [comment]
Screen Shot 2022-05-12 at 9 06 37 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**